test/SourceKit/Refactoring/semantic-refactoring/local-rename.swift

```diff
@@ -12,11 +12,24 @@ func foo() {
 }
 
 // RUN: %empty-directory(%t.result)
-// RUN: %sourcekitd-test -req=refactoring.rename.local -pos=2:8 -name new_name %s -- %s > %t.result/local-rename.swift.expected
-// RUN: %diff -u %S/local-rename.swift.expected %t.result/local-rename.swift.expected
-// RUN: %sourcekitd-test -req=find-local-rename-ranges -pos=2:8 %s -- %s > %t.result/local-rename-ranges.swift.expected
-// RUN: %diff -u %S/local-rename-ranges.swift.expected %t.result/local-rename-ranges.swift.expected
-// RUN: %sourcekitd-test -req=find-local-rename-ranges -pos=7:11 %s -- %s > %t.result/local-rename-lazy.swift.expected
-// RUN: %diff -u %S/local-rename-lazy.swift.expected %t.result/local-rename-lazy.swift.expected
+// RUN: %sourcekitd-test -req=find-local-rename-ranges -pos=2:8 -name new_name %s -- %s | %FileCheck %s --check-prefix CHECK-aa
```

Comment:
What's `name` used for?

Reply:
It’s not used at all, AFAICT. I’ll remove it.